packages/core/src/payment/strategies/googlepay/googlepay-payment-processor.ts

```diff
@@ -47,10 +48,13 @@ export default class GooglePayPaymentProcessor {
         private _requestSender: RequestSender,
     ) {}
 
-    initialize(methodId: string): Promise<void> {
+    initialize(
+        methodId: string,
+        googlePayClientOptions?: Partial<GooglePayPaymentOptions>,
```

Comment:
Why is this using an undefined and a partial interface.

Reply:
payment processor used in two cases. One i buyNow flow - then we pass googlePayClientOptions to initialize method (we use partial because in this case we don't need environment from interface). And the second flow - regular flow , so we don't pass googlePayClientOptions to initialize method

Reply:
Remover partial